src/processing_pipeline/stage_1_preprocess.py

```
@@ -76,7 +77,15 @@ def run(cls, audio_file, gemini_key):
                     ],
                 ),
             )
-            return result.text
+
+            if not result.parsed:
+                finish_reason = result.candidates[0].finish_reason
+                if finish_reason == FinishReason.MAX_TOKENS:
+                    raise ValueError("The response from Gemini was too long and was cut off.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)